runtime/build.gradle

```diff
@@ -21,7 +21,7 @@ kotlin {
         }
     }
 
-    js {
+    js(BOTH) {
```

Comment:
While for the runtime library it is enough to publish atrifacts for jsIr, the plugin needs some adjustments to be able run benchmarks